pimpmycause/pimpmycause/templates/core/contact.html

```diff
@@ -31,7 +31,7 @@ <h1 class="pnl__title">Contact Us</h1>
                                 </div>
                                 <div class="row">
                                     <div class="medium-12 large-12 small-12 columns">
-                                        <textarea name="message" rows="10" placeholder="Your message here" required></textarea>
+                                        <textarea class="message" name="message" rows="10" placeholder="Your message here" required></textarea>
```

Comment:
We only need to increase the height with with adding in more rows instead of restricting it with pixel height, the textarea is draggable by the user to any size. We only want to set the initial size it shows and the user can resize it to anything.

so just changing the rows attribute.

Also all css changes are actually done in SASS css is compiled by grunt afterwards, any changes added to a css file will be overriden next time sass is compiled. All the sass files are here: https://github.com/womenhackfornonprofits/pimp-my-cause-django/tree/master/frontend/src/sass